### ISSUE TITLE
[Accessibility][Usability]A11y_VSCode_ViewExtensions_ScreenReader: NVDA does not announce any information about the data changing dynamically in treeview.

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [Microsoft/vscode#52611]
**Environment Details:** 
VSCode Version : 1.24.1
OS Version : Win10

**Additional Details:**
ScreenReader Version: NVDA2018.2

**Repro Steps:**
1. Launch VS Code while NVDA is ON.
2. Navigate to View(Alt+V) menu option and select "Extensions" from dropdown.
3. Type some text in Search Extensions text and check screenreader is reading about it or not.

**Actual:**
NVDA does not announce any information about the data changing dynamically in treeview and also when we click on Clear extensions input button, texts in the textbox gets deleted so this information is also not being read by the screenreader.

**Expected:**
When user types something in the textbox, NVDA should announce some information that how many data been found in the treeview matching those characters. And when user clicks on Delete button, then also screenreader should throw a notification or should announce that data has got deleted.

**Recommendations:**
Refer below link which is repository of bug fixes code snippets:
https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx

**User Impact:**
User will not be able to know about the data which are found matching those characters.

Attachment for Reference: 
[A11y VSCode Extensions NVDA clear and insert data.pptx](https://github.com/Microsoft/vscode/files/2123792/A11y.VSCode.Extensions.NVDA.clear.and.insert.data.pptx)

Does this issue occur when all extensions are disabled?: Yes

**Comment by isidorn** [comment]
This feels like low hanging fruit. Upon clear, there should be an alert, and after each query is done there shuold be an alert with some general information (example "104 extensions found mathcing your query"). Assigning to July

**Comment by isidorn** [comment]
@ramya-rao-a as the other one, would be great if we tackle this in september

**Comment by ramya-rao-a** [comment]
Yes, this is for September
